src/utils/safeHref.ts

```diff
@@ -3,6 +3,7 @@ export interface URLInfo {
     sameOrigin: boolean
 }
 
+const PROTOCOL_REGEX = /^[a-z]+:/
```

Comment:
Isn't `:` valid in a path? 🤔

Reply:
good point

Reply:
Actually I don't think it will match that as a protocol the `+` should require that there is at least a singular match to the [a-z] before it finds a : I could be wrong though, I'm not great at regex